src/hooks/useScriptureSettings.tsx

```diff
@@ -131,7 +131,7 @@ export function useScriptureSettings({
     }
   }, [languageId, owner, cleanUp])
 
-  const scriptureConfig = useScriptureResources(bookId, scriptureSettings, chapter, verse, isNewTestament)
+  const scriptureConfig = useScriptureResources(bookId, languageId, owner, scriptureSettings, chapter, verse, isNewTestament)
```

Comment:
Based on this change, should the version number change to v3 since this change isn't backwards compatible?

Reply:
Good question, when I think about it, this does not have to be a breaking change.  Tweaking.